test/cpp/jit/test_custom_operators.h

```diff
@@ -138,94 +136,14 @@ void testCustomOperators() {
   }
-  {
-    auto op = createOperator(
-        "traced::op(float a, Tensor b) -> Tensor",
-        [](double a, at::Tensor b) { return a + b; });
-
-    std::shared_ptr<tracer::TracingState> state;
-    std::tie(state, std::ignore) = tracer::enter({});
-
-    Stack stack;
-    push(stack, 2.0f, autograd::make_variable(at::ones(5)));
-    op.getOperation()(stack);
-    at::Tensor output = autograd::make_variable(at::empty({}));
-    pop(stack, output);
-
-    tracer::exit({IValue(output)});
-
-    std::string op_name("traced::op");
-    bool contains_traced_op = false;
-    for (const auto& node : state->graph->nodes()) {
-      if (std::string(node->kind().toQualString()) == op_name) {
-        contains_traced_op = true;
-        break;
-      }
-    }
-    ASSERT_TRUE(contains_traced_op);
-  }
-  {
-    ASSERT_THROWS_WITH(
-        createOperator(
-            "foo::bar_with_bad_schema(Tensor a) -> Tensor",
-            [](double a, at::Tensor b) { return a + b; }),
-        "Inferred 2 argument(s) for operator implementation, "
-        "but the provided schema specified 1 argument(s).");
-    ASSERT_THROWS_WITH(
-        createOperator(
-            "foo::bar_with_bad_schema(Tensor a) -> Tensor",
-            [](double a) { return a; }),
-        "Inferred type for argument #0 was float, "
-        "but the provided schema specified type Tensor "
-        "for the argument in that position");
-    ASSERT_THROWS_WITH(
-        createOperator(
-            "foo::bar_with_bad_schema(float a) -> (float, float)",
-            [](double a) { return a; }),
-        "Inferred 1 return value(s) for operator implementation, "
-        "but the provided schema specified 2 return value(s).");
-    ASSERT_THROWS_WITH(
-        createOperator(
-            "foo::bar_with_bad_schema(float a) -> Tensor",
-            [](double a) { return a; }),
-        "Inferred type for return value #0 was float, "
-        "but the provided schema specified type Tensor "
-        "for the return value in that position");
-  }
-  {
-    // vector<double> is not supported yet.
-    auto op = createOperator(
-        "traced::op(float[] f) -> int",
-        [](const std::vector<double>& f) -> int64_t { return f.size(); });
-
-    std::shared_ptr<tracer::TracingState> state;
-    std::tie(state, std::ignore) = tracer::enter({});
-
-    Stack stack;
-    push(stack, std::vector<double>{1.0});
-
-    ASSERT_THROWS_WITH(
-        op.getOperation()(stack),
-        "Tracing float lists currently not supported!");
-
-    tracer::abandon();
-  }
-  {
-    // Try to create an op using a reserved namespace
-    ASSERT_THROWS_WITH(
-        createOperator(
-            "aten::op(float[] f) -> int",
-            [](const std::vector<double>& f) -> int64_t { return f.size(); }),
-        "Tried to register a custom operator to a reserved namespace");
-  }
 }
 
 void testCustomOperatorAliasing() {
-  RegisterOperators reg({createOperator(
```

Comment:
so we didn't consider `createOperator` as part of public API, did we? (tutorial doesn't refer to it, so should be ok)

Reply:
not that I'm aware of. @zdevito ?